pkg/network/ebpf/c/protocols/grpc/helpers.h

```diff
@@ -8,11 +8,11 @@
 #include "protocols/http2/helpers.h"
 #include "protocols/grpc/defs.h"
 
-#define GRPC_MAX_FRAMES_TO_FILTER 10
+#define GRPC_MAX_FRAMES_TO_FILTER 45
 // We only try to process one frame at the moment. Trying to process more yields
 // a verifier issue due to the way clang manages a pointer to the stack.
-#define GRPC_MAX_FRAMES_TO_PROCESS 1
-#define GRPC_MAX_HEADERS_TO_PROCESS 10
+#define GRPC_MAX_FRAMES_TO_PROCESS 10
+#define GRPC_MAX_HEADERS_TO_PROCESS 20
```

Comment:
I would expect us to have UTs for the following values in order to make sure that we actually able to process/filter those amounts. Do we have a plan to create those type of tests?

Reply:
I'll add such tests in the tracer package (where the classification tests sit)

Reply:
In a sperate PR?

Reply:
no, in that PR